.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: true
```

Comment:
Is this needed? As far as I understand, it will make the test run on VM-based infrastructure which is slower than the default container-based one.

Reply:
Yes, you are right. I forgot to remove this option. I added this option because I tried to run tests of rpmlib in Travis because I thought that this lib depends only on python-rpm (available on Ubuntu) but then I found out that this lib also depends on Koji which is not available on Ubuntu Trusty.